rulebook.tex

```diff
@@ -2680,16 +2705,25 @@ \subsubsection{Navigation Challenge}\label{sec:challenge-navigation}
 Partial points may be awarded in case only a subset of target zones were
 reached, which are accumulated if the challenge is completed fully.
 
+Alternatively, a variation of this challenge may be played where the robot needs
```

Comment:
What is the reason behind making this an alternative.
driving to machine positions vs driving to zones is not really comparable:
1. due to placing constraints it is less likely to have neighboring zones
2. 12 machine sides will cause overlaps as we cannot place 6 machines on a 5x5 field legally.
To resolve these concerns I would just commit to the change (Because actually driving to machine positions is more useful than to zones.
(Also, it just makes it harder with many minor variants to understand whats going on in challenge track)

Reply:
fair enough, will change it accordingly